README.md

```diff
@@ -27,3 +27,22 @@ In case you want to use it without SUSE Manager:
 3. At the moment `db_backend` can have only two values: `oracle` or `postgresql`.
    Therefore it will load `oraclegate` or `postgresqlgate` module. Once you want
    something else, like H2, you suppose to create `h2gate` module.
+
+## How to create a release
+
+* Update version in setup.py and src/smdba/smdba (as an example see
+https://github.com/SUSE/smdba/commit/530731003cbf3a275f7fe3bfb97b4337f975a38e
+
+* After commiting it to the master branch, create a tag and push it:
+ ```git tag X.Y.Z && git push --tags``` where X.Y.Z is the new version
+
+* Call build-src.sh to create the tarball
+
+* Update the tarball, spec file and changelog at:
```

Comment:
Is the SPEC and changelog file in this project? I'd say it's not, and if so, you can't upload them. You need to edit them.

And only now I remembered: you need to create a SR to systemsmanagement:Uyuni:Master/smdba, so you don't upload directly.

Reply:
It states "update" not "upload" ;) let me make it more clear

Reply:
True, my mistake. Still that could make some people think that `osc ci` is just fine :-D